docs/UserGuide.md

```diff
@@ -246,7 +260,7 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-<box type="warning" seamless>
+<box type="warning" seamless
```

Comment:
Is the `>` intended to be missing?

Reply:
Oh I think it got accidentally removed. Will be fixing it.